routers/api/v1/org/repo.go

```diff
@@ -0,0 +1,36 @@
+package org
+
+import (
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	api "code.gitea.io/sdk/gitea"
+)
+
+// ListRepos list all of an organization's repositories.
+func ListRepos(ctx *context.APIContext) {
+	requester := ctx.User
+	org := ctx.Org.Organization
+
+	// Find all repos a user has access to within an org.
+	reposEnv, err := org.AccessibleReposEnv(requester.ID)
```

Comment:
I will also add a test...was going to ask before PR was accepted where the appropriate place for that was :) When drone runs I notice a lot of places say they have no test files is that because they are all covered in integration tests or are those tests things that would be accepted as PRs?